docker/pravega/scripts/common.sh

```diff
@@ -36,9 +36,10 @@ add_system_property_ecs_config_uri() {
 
 # Add ECS certificates into Java truststore
 add_certs_into_truststore() {
-    CERTS=/etc/ssl/certs/java/ecs-certs/*
+    CERTS=/etc/secret-volume/ca-bundle/*
     for cert in $CERTS
     do
       yes | keytool -importcert -storepass changeit -file "${cert}" -keystore /etc/ssl/certs/java/cacerts || true
```

Comment:
Does this work with multiple certificates?  I was under the impression from other tests I've done with `keytool` that you need to specify an alias otherwise the certificate gets added as `<mykey>` which isn't a trusted entry.

Reply:
Good point. Added file name as alias to avoid the potential conflict.
(OpenJDK's default truststore also uses filename as alias)